src/modal/src/components/ModalBody.js

```diff
@@ -2,29 +2,33 @@
  * @flow
  */
 
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import type { Node } from 'react';
 
 import { BodySection } from './styled/StyledModalComponents';
 
 type Props = {
   children :Node;
+  viewportScrolling ? :boolean;
 };
 
-export default class ModalBody extends Component<Props> {
+const ModalBody = (props :Props) => {
```

Comment:
Converted to functional component to appease the lint god.

Reply:
😭